dockerfile

```
@@ -10,7 +10,7 @@ FROM base AS dbt-builder
 
 # dbt
 RUN python -m venv /usr/local/venv
-RUN /usr/local/venv/bin/pip install dbt==0.20.0
+RUN /usr/local/venv/bin/pip install dbt==0.21.0
```

Should we install the latest minor version of 0.21 ? 

```suggestion
RUN /usr/local/venv/bin/pip install "dbt=>0.21.0,<0.22.0"
```

---

Great idea! Added in next commit